src/System.Composition.AttributedModel/src/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Resources;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Security;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCulture("")]
+[assembly: NeutralResourcesLanguage("en")]
+
+[assembly: ComVisible(false)]
+[assembly: SecurityTransparent]
```

Comment:
We had problems in other libraries when [SecurityTransparent] was applied, as the rest of the libraries in corefx are the default of non-transparent, and so any attempts to use those from this (e.g. Debug.Assert) will fail.